cli/tests/unit/net_test.ts

```diff
@@ -551,7 +551,6 @@ unitTest(
   },
   async function netHangsOnClose() {
     let acceptedConn: Deno.Conn;
-    const resolvable = deferred();
```

Comment:
We should use `deferred` sparingly in the tests and only where the code does not allow awaiting a promise. The problem here is any assertion failures before `resolvable.resolve()` is called will occur in an unawaited promise... I believe that may lead to the test output being harder to understand.